src/03_video_nodes.js

```diff
@@ -300,6 +300,14 @@ Class ("paella.FlashVideo", paella.VideoElementBase,{
 	},
 
 	processEvent:function(eventName,params) {
+		// #DCE MATT-1355, Live player mute button shows muted when live stream is not muted
```

Comment:
Ok, The plugin attempt was a man in the middle to intercept the one-time flash event "loadeddata" and change the param value string from "["duration:0", "framerate:30", "currentTime:90.9", "muted:false", "paused:false", "ended:false", "volume:1", "src:""", "bytesTotal:0", "bufferedBytes:0", "bufferedTime:NaN", "videoWidth:1920", "videoHeight:540"]" 
to
 "["duration:<duration retrieved from paella.matterhorn.episode.mediapackage.duration>", "framerate:30", "currentTime:90.9", "muted:false", "paused:false", "ended:false", "volume:1", "src:""", "bytesTotal:0", "bufferedBytes:0", "bufferedTime:NaN", "videoWidth:1920", "videoHeight:540"]"

The problem is that this is to causes  the paella.FlashVideo object to call it's callReadyEvent() and we don't know what havoc that will cause. So we are just going to do a CSS fix on the mute button to show it as umuted and leave everything else the way it is. 

//		if (eventName!="loadedmetadata" && eventName!="pause" && params.duration!=0 && !this._isReady){
//			this._isReady = true;
//			this._duration = params.duration;
//			this.callReadyEvent();
//		}